geometry_msgs/PoseStamped manipulator_pose
geometry_msgs/PoseArray bag_poses
bool setup_environment
string[] bag_labels
string pickup_object
string drop_object
string action
